android/src/main/java/com/hypertrack/sdk/capacitor/HyperTrackSdkPlugin.java

```diff
@@ -12,11 +12,52 @@ public class HyperTrackSdkPlugin extends Plugin {
     private HyperTrackSdk implementation = new HyperTrackSdk();
```

Comment:
why do we need additional HyperTrackSdk layer?
If there is a need for it, what will be the responsibility of the HyperTrackSdkPlugin and HyperTrackSdk?

I have a feeling that we can simplify everything byy using just HyperTrackSdkPlugin, but maybe it can be useful in future.

Reply:
The login behide that , I want to put all the sdk logic in HyperTrackSdk like validtion and error handling , HyperTrackSdkPlugin is only responsible for call the methods and throw the success and failure.

In future we will iplemented more methods and maybe have to implements some kind of event listener,  In this case it is useful

Reply:
+